apps/dashboard/app/models/user_configuration.rb

```diff
@@ -119,7 +127,7 @@ def fetch(key_value, default_value = nil)
 
     # Returns the value if they key is present in the profile, even if the value is nil
     # This is to mimic the Hash.fetch behaviour that only uses the default_value when key is not present
-    profile_config.key?(key) ? profile_config[key] : config.fetch(key, default_value)
+    profile_config.key?(key) ? profile_config[key].freeze : config.fetch(key, default_value).freeze
```

Comment:
Not sure this is needed, I am just concerned after the side effects from the update in `OodAppGroup.order`.

I initially had:
` profile_config.key?(key) ? profile_config[key].clone : config.fetch(key, default_value).clone`
But I thought that freeze was more efficient.

Reply:
This is fine, it'll at least give us an indication up the stack if we're doing something funny.